### PR TITLE
Added ability to cut request and response bodies

### PR DESCRIPTION
**Pull request by DFSko**
resolves happy-travel/agent-app-project#963

### CODE REVIEW

File: HappyTravel.HttpRequestLogger/HttpRequestLoggingHandler.cs [hunk 1]
```diff
@@ -125,6 +128,20 @@ private static string ConvertToString(Dictionary<string, string>? dictionary, St
         }
 
 
+        private static async Task<string> ReadBytes(Stream stream, int? bytes)
```
Comment (mingazhev) [inline HappyTravel.HttpRequestLogger/HttpRequestLoggingHandler.cs:131]: Why read bytes returns a string?

Comment (mingazhev) [inline HappyTravel.HttpRequestLogger/HttpRequestLoggingHandler.cs:131]: Is it collection to name it in plural?

Comment (DFSko) [reply HappyTravel.HttpRequestLogger/HttpRequestLoggingHandler.cs]: Fixed

Comment (DFSko) [reply HappyTravel.HttpRequestLogger/HttpRequestLoggingHandler.cs]: Fixed

File: HappyTravel.HttpRequestLogger/HttpRequestLoggingHandler.cs [hunk 2]
```diff
@@ -125,6 +128,20 @@ private static string ConvertToString(Dictionary<string, string>? dictionary, St
         }
 
 
+        private static async Task<string> ReadContent(Stream stream, int? bytesLimit)
+        {
+            using var reader = new StreamReader(stream, Encoding.UTF8);
+
+            if (!bytesLimit.HasValue || stream.Length < bytesLimit)
```
Comment (mingazhev) [inline HappyTravel.HttpRequestLogger/HttpRequestLoggingHandler.cs:135]: Pattern matching could be more accurate

**Review by mingazhev — APPROVED** [review]

File: HappyTravel.HttpRequestLogger/HttpRequestLoggingHandler.cs (hunk 2 above)
Comment (DFSko) [reply HappyTravel.HttpRequestLogger/HttpRequestLoggingHandler.cs]: Fixed

**Pull request merged**